src/storage/src/hummock/store/version.rs

```diff
@@ -147,10 +140,13 @@ pub enum VersionUpdate {
 
 #[derive(Clone)]
 pub struct StagingVersion {
+    pending_imm_size: usize,
+    // newer data comes last
+    pub pending_imms: Vec<ImmutableMemtable>,
```

Comment:
nits: add some documentations to explain the difference between `uploading_imms` and `pending_imms` for someone not super familier with how event handler and uploader works.